AdobeStockAsset/Model/Asset.php

```diff
@@ -53,15 +53,21 @@ public function setId($value): void
     /**
      * @inheritdoc
      */
-    public function getMediaTypeId(): int
+    public function getMediaTypeId(): ?int
     {
-        return (int) $this->getData(self::MEDIA_TYPE_ID);
+        $mediaTypeId = $this->getData(self::MEDIA_TYPE_ID);
+
+        if (!$mediaTypeId) {
+            return null;
+        }
+
+        return (int) $mediaTypeId;
```

Comment:
What do you think about simplifying this to
```suggestion
        return ($this->getData(self::MEDIA_TYPE_ID) === null) ? null : (int) $this->getData(self::MEDIA_TYPE_ID);
```

Reply:
Hi @sivaschenko 
We have used the approach provided by @hws47a.
The method looks like the following for us:
```
public function getMediaTypeId(): ?int
{
    return $this->hasData(self::MEDIA_TYPE_ID) ? (int) $this->getData(self::MEDIA_TYPE_ID) : null;
}
```
Please, let us know if we need to change something.
Thank you!